src/README.md

```diff
@@ -0,0 +1,11 @@
+# Spatial Reference Objects
+
+A Spatial Reference System Identifier (SRID) is a unique value used to unambiguously identify projected, unprojected, and local spatial coordinate system definitions. Each projected and geographic coordinate system is defined by either a well-known ID (WKID) or a definition string (WKT).
+
+Virtually all major spatial vendors have created their own SRID implementation or refer to those of an authority, such as the European Petroleum Survey Group (EPSG), or Environmental Systems Research Institute (ESRI).
+
+We provide projection codes through their WKIDs in the `epsg` and `esri` dictionaries in `src/projection_codes.jl`. For the uninitiated, their distinction is stated in [the following response](http://gis.stackexchange.com/a/18675) from Michael D. Kennedy:
```

Comment:
Or should we make generating the strings part of the build process?  It would be easy to put your generation script into build.jl instead.